src/components/ExpandButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Typography } from '@mui/material';
+import React from 'react';
+
+export interface FoldedListboxProps {
+  s?: string;
+}
+
+// TODO: Fire toggleExpand callback.
+export const ExpandButton = ({ s }: FoldedListboxProps) => (
+  <>
+    <Typography
+      variant="h6"
+      border={'1px solid lightgrey'}
```

Comment:
I think we should use: `border="1px solid lightgrey"` unless it's a calculation, function call or a number.

Reply:
Yes, its my vscode setup that adds them auto automagically. Maybe there is an eslint option for this?

Reply:
Ok, not sure which one. Maybe it's visible when hovering the erroneous string (if the lint complains that is), then see which eslint property it wants to comply with.